### ISSUE TITLE
COPY ./netboot fails

### ISSUE THREAD

**Issue by chelming** [MayankTahil/rancheros-pxe-bootstap#1]
There's no netboot folder for ipxe. What am I missing?

**Comment by MayankTahil** [comment]
I think the example directory needs to be renamed or dockerfile needs
updating but the files for TFTP for example are in that directory. I havn't
touched this project in over a year so it's me just guessing.

Sent via clumbsy thumbs

Mayank Tahilramani

**Comment by chelming** [comment]
I _think_ the installer.pxe needs to be _built_, right? I don't think I can just tftp serve up that file and run it. (I literally have never touched ipxe before today)

**Comment by chelming** [comment]
yeah, I had to compile (?) the installer.pxe to get it to work: `docker run -it -v $(pwd)/ipxe/netboot:/custom-pxe -e PXE_FILE=installer.pxe vtajzich/ipxe` and use the resulting undionly.kpxe

**Comment by jhughes2112** [comment]
Is there any reason to do the COPY command in the Dockerfile at all, is there, since the docker-compose.yaml mounts a folder over it anyway?  The host mounted folder hides whatever was in the docker image, so I imagine it would be sufficient to put the built .pxe file in there on the host.

Thanks to you both for the clues to putting this together, btw.
